marketplace/plugins/pocketbase/lib/operations.json

```diff
@@ -0,0 +1,267 @@
+
+{
+  "$schema": "https://raw.githubusercontent.com/ToolJet/ToolJet/develop/plugins/schemas/operations.schema.json",
+  "title": "PocketBase datasource",
+  "description": "A schema defining PocketBase datasource",
+  "type": "api",
+  "defaults": {},
+  "properties": {
+    "operation": {
+      "label": "Operation",
+      "key": "operation",
+      "type": "dropdown-component-flip",
+      "description": "Single select dropdown for operation",
+      "list": [
+        {
+          "value": "list_records",
+          "name": "List Records"
+        },
+        {
+          "value": "get_record",
+          "name": "Get Record"
+        },
+        {
+          "value": "add_record",
+          "name": "Add Record to Collection"
+        },
+        {
+          "value": "update_record",
+          "name": "Update Record to Collection"
+        },
+        {
+          "value": "delete_record",
+          "name": "Delete Record"
+        }
+      ]
+    },
+    "list_records": {
+      "collectionId": {
+        "label": "Collection ID or Name",
+        "key": "collectionId",
+        "type": "codehinter",
+        "lineNumbers": false,
+        "description": "Enter collection id or name",
+        "width": "320px",
+        "height": "36px",
+        "className": "codehinter-plugins col-6",
+        "placeholder": "Enter collection id or name"
+      },
+      "limit": {
+        "label": "Limit",
+        "key": "limit",
+        "type": "codehinter",
+        "lineNumbers": false,
+        "description": "Enter limit to get max records",
+        "width": "320px",
+        "height": "36px",
+        "className": "codehinter-plugins",
+        "placeholder": "Enter limit to get max records"
+      },
+      "sort": {
+        "label": "Sort",
+        "key": "sort",
+        "type": "codehinter",
+        "lineNumbers": false,
+        "description": "Enter records sort rule in order",
+        "width": "320px",
+        "height": "36px",
+        "className": "codehinter-plugins",
+        "placeholder": "Enter record sort rule in order (Prefix '-' to field for descending order)"
+      },
+      "filterField": {
```

Comment:
@akshaysasidrn Also I was thinking, instead of adding codehinter can we add this
![pocketbase-filter](https://github.com/ToolJet/ToolJet/assets/111295371/dd685b1e-a3f3-46f8-bd48-887afca354a7)
I recently implemented this for supabase plugin. It will align with tooljet filter UI and can add multiple filter conditions as well. Tooljet DB datasource also has this implementation.
In terms of effort, codehinter gonna have less effort.
Let me know your thoughts on this then I'll move forward with changes.

Reply:
It'd be great if you can implement that. It goes with the same design language as the rest of the data sources.

Reply:
Done.
![pocketbase-filters](https://github.com/ToolJet/ToolJet/assets/111295371/bf659032-8232-437b-81e1-fe24d05edc74)